File: native/mixdog-graph/tests/fixtures/calls/lua/sample.lua
-- hidden()
quoted = "hidden()"

function inner() end
function leaf(x) return x end
function nest(x) end
function helper() end
function seed() end
function plain() end

plain()

function run(a)
  inner()
  nest(leaf(1))
  a.b().c()
end

Widget = {}
function Widget:ping() end
function Widget:act()
  helper()
  self:ping()
end
function Widget.new()
  return Widget
end

mark = "μ"; seed()
Widget.new()

File: native/mixdog-graph/tests/fixtures/calls/objc/sample.m
#import <Foundation/Foundation.h>

/* hidden() */
static const char *quoted = "hidden()";

static int inner(void) { return 0; }
static int leaf(int x) { return x; }
static int nest(int x) { return x; }
static int helper(void) { return 0; }
static int seed(void) { return 0; }
static int plain(void) { return 0; }

int g = plain();

int run(void) {
  inner();
  nest(leaf(1));
  return 0;
}

@interface Widget : NSObject
- (void)ping;
- (void)act;
@end

@implementation Widget
- (void)ping {}
- (void)act {
  id a = nil;
  helper();
  [self ping];
  [[a b] c];
}
@end

int seed_line(void) {
  const char *mark = "μ"; seed();
  [Widget new];
  return 0;
}
